javascript/index.js

```diff
@@ -48,18 +80,63 @@ function setSplitBtn() {
 
 function setStartBtn() {
   // ... your code goes here
+  
 }
 
 function setResetBtn() {
   // ... your code goes here
 }
 
 // Start/Stop Button
-btnLeft.addEventListener('click', () => {
-  // ... your code goes here
-});
+
+ document.getElementById('btnLeft').addEventListener("click", () => {
+  const btnLeftClassName = document.getElementById('btnLeft').className;
+  if(btnLeftClassName==="btn start"){
+   chronometer.startClick(printTime);
+  btnLeft.innerHTML="STOP";
+  btnLeft.className="btn stop"
+  btnRight.className="btn split"
+  btnRight.innerHTML="SPLIT"
+  chronometer.stopClick()
+  }
+
+ if(btnLeftClassName==="btn stop"){
+  chronometer.stopClick()
+  btnLeft.innerHTML="STOP";
```

Comment:
That's incorrect, after clicking on "STOP" you want the button to turn back to "START"
```
btnLeft.className = 'btn start';
btnLeft.textContent = 'START';